config/crd/standard/gateway.networking.k8s.io_safeUpgradesVAP.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes-sigs/gateway-api/pull/3328
+    gateway.networking.k8s.io/bundle-version: v1.4.0
+    gateway.networking.k8s.io/channel: standard
```

Comment:
I'm not 100% sure if this should be part of a release channel or not. Fine for now, but may want to adjust if we make a VAP that is meant to be included with both release channels.